src/current/_includes/v23.2/known-limitations/row-level-ttl-limitations.md

```diff
@@ -1 +1,4 @@
 - Any queries you run against tables with Row-Level TTL enabled do not filter out expired rows from the result set (this includes [`UPDATE`s]({% link {{ page.version.version }}/update.md %}) and [`DELETE`s]({% link {{ page.version.version }}/delete.md %})). This feature may be added in a future release. For now, follow the instructions in [Filter out expired rows from a selection query]({% link {{ page.version.version }}/row-level-ttl.md %}#filter-out-expired-rows-from-a-selection-query).
+- Enabling Row-Level TTL on a table with multiple [secondary indexes]({% link {{ page.version.version }}/indexes.md %}) can have negative performance impacts on a cluster, including increased [latency]({% link {{ page.version.version }}/common-issues-to-monitor.md %}#service-latency) and [contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention). This is particularly true for large tables (with terabytes of data and billions of rows). 
```

Comment:
could we change 

> with terabytes of data and billions of rows

with terabytes of data and billions of rows, split up into multiple ranges & on separate nodes

or something of the sort

Reply:
Updated to include that info - please take a look!